packages/@lwc/compiler/src/transformers/__tests__/transform.spec.ts

```diff
@@ -6,7 +6,7 @@
  */
 import { transform, transformSync } from '../../transformers/transformer';
 
-function testValidateOptions(methodName, method) {
+function testValidateOptions(methodName: string, method: any) {
```

Comment:
I'll be the first to admit that I'm not a TypeScript expert. So I added a lot of `as any` to these files to make them valid TypeScript. I'm happy to take other suggestions.